snippets/man_gen/hubl_blog_variables.json

```diff
@@ -4,20 +4,25 @@
     "body": [
       "{{ ${1|blog_author,blog_author.avatar,blog_author.bio,blog_author.display_name,blog_author.email,blog_author.facebook,blog_author.google_plus,blog_author.has_social_profiles,blog_author.linkedin,blog_author.twitter,blog_author.website|} }}"
     ],
-    "description": "Data about blog authors. To be used for blog author listing pages ({% if blog_author %})."
+    "description": "This variable contains blog author information for blog author listing pages. It can be used to create conditional logic to >render markup for blog author listings ({% if blog_author %})."
```

Comment:
```suggestion
    "description": "This variable contains blog author information for blog author listing pages. It can be used to create conditional logic to render markup for blog author listings ({% if blog_author %})."
```